src/com/dotcms/rest/api/v1/authentication/ForgotPasswordResource.java

```diff
@@ -47,30 +48,26 @@
     private final UserManager userManager;
     private final CompanyAPI  companyAPI;
     private final AuthenticationHelper  authenticationHelper;
-    private final SecurityLoggerServiceAPI securityLogger;
```

Comment:
The security logger service api is a wrapper of the SecurityLogger, it is useful just as a mock, since the SecurityLogger can not be tested.